.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -397,6 +402,14 @@ jobs:
           python3 -mpip install codecov
           python3 -mcodecov
       {%- endif %}
+      - uses: actions/upload-artifact@v2
+        name: Upload crash reports
+        if: always()
+        with:
+          name: crash-reports-${{ matrix.config }}
+          retention-days: 14
+          path:
+            pytorch_crashes/*.dmp
```

Comment:
Pre-optimization but should we try and put all of the dump files into a single archive?

We ran into issues in the past with trying to upload a ton of files at once, see https://github.com/pytorch/pytorch/pull/60475

Reply:
It should kill the test process so there's only going to be one, we just don't know the name ahead of time